_episodes/07-templates.md

```diff
@@ -17,6 +17,41 @@ keypoints:
 - "You can avoid duplicated effort by basing new layouts on previous ones"
 ---
 
+> ## Expanding the layout
+>
+> We will probably want to include the contact line we added in the previous section
+> in every standard page on our site.
+> It would be better placed into the layout template,
+> so we don't have to remember to `include` it every time we make a new page.
+> Create a new layout, `page-contact.html`,
+> which is based on the existing `page` layout and
+> displays the contact information from `_includes/contact.md`
+> after the page content.
+> Check that this works by applying that layout to the `team.md` page.
+>
+> > ## Solution
+> > Your new layout file, `_layouts/page-contact.html`,
+> > should look like this:
+> >
+> > ~~~
+> > ---
+> > layout: page
+> > ---
```

Comment:
```suggestion
> > ---
> > 
```
Sometimes issues with rendering when there is no blank line after the header.

Reply:
well spotted, thanks!